resources/assets/js/annotations/annotatorContainer.vue

```diff
@@ -156,8 +156,12 @@ export default {
             return imagesIds;
         },
         hasCrossOriginError() {
-            return !this.loading && this.crossOriginError;
+            return !this.loading && this.crossOriginError && !this.crossOriginTiffError;
         },
+        hasCrossOriginErrorTiff() {
+            return this.crossOriginError && this.crossOriginTiffError;
```

Comment:
```suggestion
            return this.loading && this.crossOriginTiffError;
```


I think you can replace `crossOriginError` by `loading`, because 
1. `crossOriginTiffError` is independent of `crossOriginError` and 
2. the error message should only be shown when it finished loading